client/lib/dsp/README.md

```diff
@@ -0,0 +1,3 @@
+# DSP Widget
+
+
```

Comment:
Sorry if this is a silly question, but what does DSP stand for? Can we use a more descriptive name?

Reply:
I was being silly! I've updated it with a bit more info, but it's likely we'll update it again along the way. Thanks for looking at this @bluefuton , appreciate it!